workflow-testing/src/test/java/com/squareup/workflow/testing/WorkerRenderExpectationsTest.kt

```diff
@@ -0,0 +1,234 @@
+package com.squareup.workflow.testing
+
+import com.squareup.workflow.Worker
+import com.squareup.workflow.Workflow
+import com.squareup.workflow.WorkflowOutput
+import com.squareup.workflow.action
+import com.squareup.workflow.runningWorker
+import com.squareup.workflow.stateless
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlin.reflect.typeOf
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+// TODO add more failure tests
```

Comment:
Want to track this and get this merged?

Reply:
Yea, got a few other PRs stacking up on this. I'll file.

Reply:
#117